src/elements-experimental/seat-reservation/seat-reservation-area/seat-reservation-area.scss

```
@@ -0,0 +1,46 @@
+@use '@sbb-esta/lyne-elements/core/styles' as sbb;
```

```suggestion
@use '@sbb-esta/lyne-elements/core/styles' as sbb;

// Box-sizing rules contained in typography are not traversing Shadow DOM boundaries. We need to include box-sizing mixin in every component.
@include sbb.box-sizing;
```

I recommend to do it in all scss files (we do it). then manually assigning box-sizing is obsolete